source/code-snippets/usage-examples/bulkWrite.js

```diff
@@ -40,19 +44,22 @@ async function run() {
         },
       },
       {
+        // Update documents that match the specified filter
         updateMany: {
           filter: { "location.address.zipcode": "44011" },
           update: { $set: { is_in_ohio: true } },
           upsert: true,
         },
       },
       {
+        // Delete a document that matches the specified filter
         deleteOne: { filter: { "location.address.street1": "221b Baker St" } },
       },
     ]);
-
+    // Log the result of the bulk write operation 
     console.log(result);
   } finally {
+    // Close the database connection when the operations are completed or if an error occurs
```

Comment:
S: I have to use the horizontal scroll to see this full comment at the staging link, maybe change to a multi-line comment